packages/Filters/src/types.js

```diff
@@ -0,0 +1,87 @@
+export const rules = {
```

Comment:
`rules`, `defaultRulesByType` and `localeKeysByRule ` are duplicated in `types.js` and `rules.js`... think its better to extract?

Reply:
This is a mistake for sure!